packages/abi-gen/src/index.ts

```diff
@@ -118,6 +105,13 @@ for (const abiFileName of abiFileNames) {
         process.exit(1);
     }
 
+    const outFilePath = `${args.output}/${utils.makeOutputFileName(namedContent.name)}.ts`;
```

Comment:
Let's break this into two lines for readability: 

```
const fileName = utils.makeOutputFileName(namedContent.name);
const outFilePath = `${args.output}/${fileName}.ts`;
```

Reply:
Thanks for this suggestion.  The length of this line irked me too.  I originally wrote it as
```
const outFilePath =
    `${args.output}/${utils.makeOutputFileName(namedContent.name)}.ts`;
```
but `prettier:ci` wouldn't let me keep it that way.